### PR TITLE
Yel 7817 upgrade tf1.5

### PR DESCRIPTION
**Pull request by alastairallan**

### CODE REVIEW

**Review by PeterMillerHO — APPROVED** [review]
LGTM

File: .drone.yml [hunk 1]
```diff
@@ -77,6 +79,6 @@ steps:
     event:
       - push
 
-services:
-  - name: docker
-    image: *acp-docker-image
\ No newline at end of file
+#services:
+#  - name: docker
+#    image: *acp-docker-image
```
Comment (ianrelf) [inline .drone.yml:84]: It is better to remove this completely, rather than comment it out.

File: .drone.yml [hunk 2]
```diff
@@ -35,7 +35,9 @@ x-anchors:
     - sleep 20
 
   check-format: &check-format
-    - docker run --rm -v $(pwd):/data -w /data hashicorp/terraform fmt --diff --check
+    - terraform --version
+    - terraform fmt --diff --check
+#    - docker run --rm -v $(pwd):/data -w /data hashicorp/terraform fmt --diff --check
```
Comment (ianrelf) [inline .drone.yml:40]: It is better to remove this completely, rather than comment it out.

File: .drone.yml [hunk 3]
```diff
@@ -35,7 +35,9 @@ x-anchors:
     - sleep 20
```
Comment (ianrelf) [inline .drone.yml:35]: Also, this sleep can now be removed.

**Review by ianrelf — CHANGES_REQUESTED** [review]

File: .drone.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 global-variables:
-  acp-docker-image: &acp-docker-image 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind:latest
+#  acp-docker-image: &acp-docker-image 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind:latest
```
[the remark this answers is not in the document]
Comment (alastairallan) [reply .drone.yml]: Removed

File: .drone.yml (hunk 2 above)
Comment (alastairallan) [reply .drone.yml]: Removed

**Review by ianrelf — APPROVED** [review]
LGTM

**Pull request merged**